.github/workflows/build-git-installers.yml

```diff
@@ -211,11 +211,22 @@ jobs:
           sed -i -e \
             's/recently_seen="\$(get_recently_seen)"/recently_seen="bogus-to-keep-pestering"/' \
             $b/git-extra/git-update-git-for-windows
-      - name: Set the installer Publisher to the Git Fundamentals team
+      - name: Update installer metadata for microsoft/git
         shell: bash
         run: |
           b=/usr/src/build-extra &&
-          sed -i -e 's/^\(AppPublisher=\).*/\1The Git Fundamentals Team at GitHub/' $b/installer/install.iss
+          # Remove '.vfs' from version
+          version=$(echo ${{ needs.prereqs.outputs.tag_version }} | sed s/.vfs//g) &&
+          # Replace AppPublisher and AppVersion with correct values for microsoft/git
+          sed -i -e "s/^\(AppPublisher=\).*/\1Microsoft/" $b/installer/install.iss &&
```

Comment:
Should this be "Microsoft Corporation" (per your linked installer metadata spec)?

Reply:
Thanks for pointing this out! As I was experimenting with `wingetcreate`, I realized that it joins the Publisher to the PackageName, separated by a dot with whitespace removed. There could be a world where we try to override this in the `wingetcreate` code, but I wanted to prioritize getting an MVP out then making adjustments later if needed. I've updated the issue to align with the actual expectations for manifest data with these updates.